### PR TITLE
Update contributing.md

### PR DESCRIPTION
**Pull request by krlv**
1. Add composer scripts to run `phpunit` and `php-cs-fixer`
2. Update `CONTRIBUTING.md`: add testing section; add composer command to code style section

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,4 +27,18 @@ The amphp project adheres to the PSR-2 style guide with the exception that
...
+
+To apply amphp code standards you can run `php-cs-fixer` with following composer command:
+
+```bash
+composer code-style
+```
+
+## Running the tests
+
+Run the test suite from amphp root directory:
```
Comment (kelunik) [inline CONTRIBUTING.md]: I'd remove the `amphp` here.

**Comment by kelunik** [comment]
@bwoebi Please use the `rebase` or `squash` option next time for merging, so we end up with a cleaner commit history.

**Comment by bwoebi** [comment]
@kelunik I usually squash merge if 3 or more commits. And generally do not rebase PRs.

**Comment by kelunik** [comment]
Why not?

**Pull request merged**